src/virtac/atip_ioc_entry.py

```diff
@@ -20,21 +20,29 @@
 def parse_arguments():
     """Parse command line arguments sent to virtac"""
     parser = argparse.ArgumentParser()
-    parser.add_argument("ring_mode", nargs="?", type=str, help="Ring mode name")
     parser.add_argument(
-        "--disable-emittance",
+        "ring_mode",
+        nargs="?",
+        type=str,
+        help="The ring mode to be used, e.g., IO4 or DIAD",
+    )
+    parser.add_argument(
         "-d",
-        help="disable the simulator's time-consuming emittance calculation",
+        "--disable-emittance",
+        help="Disable the simulator's time-consuming emittance calculation",
         action="store_true",
     )
     parser.add_argument(
-        "--enable-tfb",
         "-t",
-        help="simulate extra dummy hardware to be used by the Tune Feedback system",
+        "--enable-tfb",
+        help="Simulate extra dummy hardware to be used by the Tune Feedback system",
         action="store_true",
     )
     parser.add_argument(
-        "--verbose", "-v", help="increase output verbosity", action="store_true"
+        "-v",
+        "--verbose",
+        help="Increase output and logging verbosity",
```

Comment:
What is 'output verbosity'? I can only see a logging verbosity increase.

Reply:
Yep it only does logging verbosity, I'll remove "output and"